src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -465,9 +466,16 @@ static int open_device(const char* path, bool try_load_module = false)
     if (r < 0) {
       cerr << "rbd-nbd: failed to load nbd kernel module: " << cpp_strerror(-r) << std::endl;
       return r;
+    } else {
+      load_module = true;
     }
     nbd = open(path, O_RDWR);
   }
+
+  if (nbds_max && try_load_module && !load_module) {
+    cerr << "rbd-nbd: nbds_max param doesn't work, because nbd module already loaded." << std::endl;
```

Comment:
@liupan1111

I would prefer a message like "rbd-nbd: ignoring nbds_max parameter: nbd module already loaded" or "rbd-nbd: nbd module already loaded, ignoring nbds_max parameter". Or may be Jason will have a better suggestion as a native speaker.

Also, don't we have to do the same for `max_part`?